src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: null,
```

Comment:
usually we set value of same type with falsy value here

for example empty string, no need to set null for every type